src/components/scatterplot/ScatterplotSubscriber.js

```diff
@@ -171,15 +202,21 @@ export default function ScatterplotSubscriber(props) {
         cellSelection={cellSelection}
         cellHighlight={cellHighlight}
         cellColors={cellColors}
+        cellSetPolygons={cellSetPolygons}
+
+        cellSetLabelSize={cellSetLabelSize}
+        cellSetLabelsVisible={cellSetLabelsVisible}
+        cellSetPolygonsVisible={cellSetPolygonsVisible}
 
         setCellFilter={setCellFilter}
         setCellSelection={setCellSelectionProp}
         setCellHighlight={setCellHighlight}
-        cellRadiusScale={cellRadiusScale}
+        cellRadiusScale={cellRadiusScale * cellRadius}
```

Comment:
What happened here?

Reply:
We already had `cellRadiusScale` to deal with differences in the ratio of cell radii to plot extent. Now that we are adding the cell radius slider in the UI, we still want to respect that `cellRadiusScale` value which may have been computed at the plot's initialization